Packs/SuspiciousDomainHunting/Playbooks/playbook-Suspicious_Domain_Hunting_Incident_Handling.yml

```diff
@@ -0,0 +1,1330 @@
+    type: regular
+    task:
+      id: 03987393-00ff-422c-820e-bd371bea37e1
+      version: -1
+      name: Rasterize Original Webpage
+      description: Converts the contents of a URL to an image file or a PDF file.
+      tags:
+      - Domain_rasterize
+      script: '|||rasterize'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "26"
+    scriptarguments:
+      file_name:
+        simple: original
+      url:
+        complex:
+          root: incident
+          accessor: certstreamuserasset
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 202.5,
+          "y": -100
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: 27d16b99-15ce-4f93-84e9-f77f6afc1b84
+    type: condition
+    task:
+      id: 27d16b99-15ce-4f93-84e9-f77f6afc1b84
+      version: -1
+      name: Is whois Integration Available?
+      description: Returns 'yes' if integration brand is available. Otherwise returns 'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "27"
+      "yes":
+      - "22"
+    scriptarguments:
+      brandname:
+        simple: Whois
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 817.5,
+          "y": -750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "30":
+    id: "30"
+    taskid: fbe4aed7-5138-496a-83eb-6ecca2b96dd5
+    type: title
+    task:
+      id: fbe4aed7-5138-496a-83eb-6ecca2b96dd5
+      version: -1
+      name: Benign
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "31"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1700,
+          "y": 985
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: f1ab5ead-ecb2-4b5a-8ecc-13a25e550ddd
+    type: regular
+    task:
+      id: f1ab5ead-ecb2-4b5a-8ecc-13a25e550ddd
+      version: -1
+      name: Set closing reason as Benign
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "34"
+    scriptarguments:
+      key:
+        simple: Verdict
+      value:
+        simple: Benign
```

Comment:
Please change the key value to 'Non-Malicious'. This key is supposed to be presented in the layout using a new script in this PR. However, there is already a 'verdictResult' script that was merged a few months ago and is currently located under common scripts. The correct script should present the verdict as 'Non-Malicious' instead of 'Benign'.